msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-meeting-stage.md

```diff
@@ -0,0 +1,196 @@
+
+1. In the app gallery, select the app that you want to add and follow the steps as required. After adding the tab, you can view it as.
+
+   :::image type="content" source="../assets/images/side-panel-view.png" alt-text="This screenshot shows side panel view with the list of apps.":::
+
+### Meeting stage view
+
+1. Once a tab is added to the meeting side panel, you can now choose to opt into global app sharing.
+
+1. This results in rendering tab on the stage for every participant in the meeting.
+
+   :::image type="content" source="../assets/images/meeting-stage-view.png" alt-text="This screenshot shows meeting stage view of the app you shared to meeting.":::
+
+### Apps in channel meeting
+
+A public scheduled channel meeting has the same list of apps as its parent team. Installing an app to a channel meeting also makes it available in the parent team, and vice versa.
+
+However, the tab instances in a channel meeting are separate from the tabs in the channel itself. For example, suppose a "Development" channel has a "Polly" tab. If you create a "Standup" meeting in that channel, that meeting would not have a "Polly" tab, until you explicitly [add the tab to the meeting](#meeting-details-view).
+
+In public scheduled channel meetings, after a meeting tab is added it can be accessed from the meeting details page by selecting on the meeting object.
+
+:::image type="content" source="~/assets/images/apps-in-meetings/after-a-meeting1.png" alt-text="After a meeting":::
+
+> [!NOTE]
+> On mobile, anonymous users will not be able to access apps in scheduled public channel meetings.
+
+### App manifest settings for Tabs in meeting
+
+Update your [app manifest](/microsoftteams/platform/resources/schema/manifest-schema) with fine grained context prop to target the above different tab views. The meetings app capabilities are declared in your app manifest using the scopes and context arrays under configurableTabs section.
+
+#### Scope
+
+The scope defines who can access the apps.
+
+* `groupchat` scope makes your app available in group scope and enables app to be added in a call or meeting (scheduled private meeting/ instant meetings).
+
+* `team` scope makes your app available in team scope and enables your app to be added in team or channel or scheduled channel meeting.
+
+#### Context
+
+The `context` property determines if the app is available in specific view after installation and configuration. Following are the values for the `context` property from which you can use all or some of the values:
+
+|Value|Description|
+|---|---|
+| **channelTab** | A tab in the header of a team channel. |
+| **privateChatTab** | A tab in the header of a group chat between a set of users, not in the context of a team or meeting. |
+| **meetingChatTab** | A tab in the header of a group chat between a set of users for a scheduled meeting. You can specify either **meetingChatTab** or **meetingDetailsTab** to ensure the apps work in mobile. |
```

Comment:
`You can specify either **meetingChatTab** or **meetingDetailsTab** to ensure the apps work in mobile.` Where on mobile do these context work? Does the user need to add these contexts for mobile meeting users, or is it just the pre/post meeting tab?

Reply:
Also I'm not sure how to explain it but this sentence reads a bit odd, perhaps it needs to be a bit more definitive.
"To ensure the app works on mobile specify either **meetingChatTab** or **meetingDetailsTab** context".